### PR TITLE
Changes in rali unittests

### PR DESCRIPTION
**Pull request by Indumathi31**

### CODE REVIEW

File: rocAL/rocAL/include/caffe_meta_data_reader_detection.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ public :
 private:
     void read_files(const std::string& _path);
     bool exists(const std::string &image_name);
-    void add(std::string image_name, BoundingBoxCords bbox, BoundingBoxLabels b_labels,ImgSizes image_size);
+    void add(std::string image_name, BoundingBoxCords bbox, BoundingBoxLabels b_labels,ImgSize image_size);
```
Comment (shobana-mcw) [inline rocAL/rocAL/include/caffe_meta_data_reader_detection.h:52]: Space after comma

File: rocAL/rocAL/include/coco_meta_data_reader.h [hunk 2]
```diff
@@ -38,18 +38,18 @@ class COCOMetaDataReader: public MetaDataReader
...
     BoundingBoxBatch* _output;
     std::string _path;
     int meta_data_reader_type;
-    void add(std::string image_name, BoundingBoxCords bbox, BoundingBoxLabels b_labels, ImgSizes image_size);
+    void add(std::string image_name, BoundingBoxCords bbox, BoundingBoxLabels b_labels, ImgSize image_size);
     bool exists(const std::string &image_name);
     std::map<std::string, std::shared_ptr<BoundingBox>> _map_content;
     std::map<std::string, std::shared_ptr<BoundingBox>>::iterator _itr;
-    std::map<std::string ,std::vector<ImgSize>> _map_img_sizes;
+    std::map<std::string ,ImgSize> _map_img_sizes;
```
Comment (shobana-mcw) [inline rocAL/rocAL/include/coco_meta_data_reader.h:52]: space after comma

File: rocAL/rocAL/include/coco_meta_data_reader.h [hunk 3]
```diff
@@ -38,18 +38,18 @@ class COCOMetaDataReader: public MetaDataReader
     void print_map_contents();
     bool set_timestamp_mode() override { return false; }
     MetaDataBatch * get_output() override { return _output; }
-    std::map<std::string, std::shared_ptr<BoundingBox>> get_map_content() { return _map_content;}
+    const std::map<std::string, std::shared_ptr<BoundingBox>> & get_map_content() { return _map_content;}
```
Comment (shobana-mcw) [inline rocAL/rocAL/include/coco_meta_data_reader.h:41]: check in coding guidelines or in other instances the spacing for & (reference)

File: rocAL/rocAL/include/meta_data.h [hunk 4]
```diff
@@ -49,12 +49,12 @@ struct MetaData
     BoundingBoxCords& get_bb_cords() { return _bb_cords; }
     BoundingBoxCords_xcycwh& get_bb_cords_xcycwh() { return _bb_cords_xcycwh; }
     BoundingBoxLabels& get_bb_labels() { return _bb_label_ids; }
-    ImgSizes& get_img_sizes() {return _img_sizes; }
+    ImgSize& get_img_size() {return _img_size; }
```
Comment (shobana-mcw) [inline rocAL/rocAL/include/meta_data.h:52]: Space after open parenthesis.

File: rocAL/rocAL/include/meta_data.h [hunk 5]
```diff
@@ -72,27 +72,27 @@ struct BoundingBox : public MetaData
         _bb_cords =std::move(bb_cords);
         _bb_label_ids = std::move(bb_label_ids);
     }
-    BoundingBox(BoundingBoxCords bb_cords,BoundingBoxLabels bb_label_ids ,ImgSizes img_sizes)
+    BoundingBox(BoundingBoxCords bb_cords,BoundingBoxLabels bb_label_ids ,ImgSize img_size)
```
Comment (shobana-mcw) [inline rocAL/rocAL/include/meta_data.h:75]: Adjust spacing for this constructor arguments

**Review by shobana-mcw — COMMENTED** [review]
I am finding spacing issues in most of the places where img size vector modification is made. Change that.

**Comment by Indumathi31** [comment]
Sure Shobana. I will make these changes. 
Cherry-picked working commit of Sampath. I will push the cleaned codes.

**Pull request closed without merging**